client/db.go

```diff
@@ -374,8 +376,14 @@ func (db *DB) Run(b *Batch) error {
 //
 // TODO(pmattis): Allow transaction options to be specified.
 func (db *DB) Tx(retryable func(tx *Tx) error) error {
+	_, file, line, ok := runtime.Caller(1)
```

Comment:
Nice, thanks for checking that. The equivalent call was reasonably cheap in MRI but very expensive in JRuby.